charts/k8ssandra/Chart.yaml

```diff
@@ -25,11 +25,9 @@ version: 0.25.0
 appVersion: 3.11.7
 dependencies:
   - name: cass-operator
-    version: 1.5.0
-    # The cass-operator chart lives in https://github.com/datastax/cass-operator. There are
-    # changes in master that are needed for K8C-19. For now we will sync (i.e., copy) the
-    # cass-operator chart into the charts subdirectory.
-    #repository: https://github.com/datastax/cass-operator/tree/master/charts
+    version: 0.25.0
```

Comment:
Separate discussion, but related to yesterday's meeting. If we split to sub charts, do they have to always follow the same version as parent? I'd like to separate so it becomes easier to track subchart version history. We can discuss in Slack.

Reply:
At the moment, yes the charts need to have the same version. We obviously need to change that. And as we start to introduce more charts, I think changing it becomes a bigger priority.